app/src/main/res/values/styles.xml

```diff
@@ -337,4 +179,20 @@
         <item name="android:windowAnimationStyle">@null</item>
     </style>
 
+    <style name="ToolbarTheme" parent="ThemeOverlay.AppCompat.Dark.ActionBar">
+        <item name="android:textColorPrimary">@android:color/white</item>
+        <item name="android:textColorSecondary">@android:color/white</item>
+        <item name="popupTheme">@style/ThemeOverlay.AppCompat.DayNight.ActionBar</item>
+        <item name="tint">@color/white</item>
+        <item name="iconTint">@color/white</item>
+        <item name="android:textCursorDrawable">@drawable/white_edit_text_cursor</item>
```

Comment:
Question: ain't there a way to change the color of the cursor without introducing a new variable?

Reply:
I remember it depends on accent color, I will try changing it for the container